With the rising need for democratizing sustainability and health, many new technology developments focus on the efficient utilization of large-scale biological data. Unlike datasets close related to our daily activities (i.e. photos, texts), large-scale biological datasets (i.e. omics) often require extensive domain-specific knowledge to acquire and interpret. Among these omics datasets, feature extractions from sequential data (e.g., genomics, transcriptomics, proteomics) have a proven record to benefit from machine learning. However, interpretable representations of the chemical fingerprints resulting from the aforementioned sequential data (i.e. metabolomics) by probabilistic models are still challenging. Here, this project takes an empirical approach to represent a set of metabolomics data with existing probabilistic modeling tools. Second-line evidence of biological phenotypes or sequential omics data may be introduced to evaluate the interpretability of the representations. 
